### PR TITLE
manage_vhost sometimes breaks and redis version update results a fatal error during deploy

### PR DESCRIPTION
**Pull request by stijnbernards**
This MR solves 2 things:

1. `manage_vhost` is ran before the current symlink has been updated. This results in errors during nginx setup and breaks automatic deployments. It also seems that the error which this may result fully stops the automatic nginx reload
2.  `hypernode-systemctl settings redis_version {$config->getVersion()} --block"` asks for a yes/no input. Which is not provided by deploy. If redis is upgraded to 7.x during deploy the deployment will never finish since the prompt will always come up even if redis is already on version 7.x

### CODE REVIEW

File: src/Deployer/Task/PlatformConfiguration/NginxTask.php [hunk 1]
```diff
@@ -37,13 +37,14 @@ public function configureWithTaskConfig(TaskConfigurationInterface $config): ?Ta
 
         task('deploy:nginx', [
```
Comment (tdgroot) [inline src/Deployer/Task/PlatformConfiguration/NginxTask.php:38]: So now it first places nginx files and after that it runs HMV. Sounds OK to me, but what if the vhost doesn't exist on the node yet? Have you tested that scenario as well?

**Review by tdgroot — COMMENTED** [review]
Hey @stijnbernards! Thanks for your patches, much appreciated!

File: src/Deployer/Task/PlatformConfiguration/NginxTask.php (hunk 1 above)
Comment (stijnbernards) [reply src/Deployer/Task/PlatformConfiguration/NginxTask.php:38]: Not yet, but I believe the deploy creates the www.shop.nl folders in the nginx directory aswell, not sure if creating a new vhost will then remove this. Let me see if I can test this somewhere :)

Comment (tdgroot) [reply src/Deployer/Task/PlatformConfiguration/NginxTask.php:38]: Could you verify this or drop this part of the patch? Then we can merge!

Comment (stijnbernards) [reply src/Deployer/Task/PlatformConfiguration/NginxTask.php:38]: I'll try and verify this change soon but it's the most important fix of this MR. I'll create a new MR to fix the hypernode settings

Comment (stijnbernards) [reply src/Deployer/Task/PlatformConfiguration/NginxTask.php:38]: Just verified this change and it does infact fix my issue when creating an entirely new vhost during deploys when it does not exist yet. Same goed for the deploy:varnish:activate step as this step uses the current folder

**Review by tdgroot — APPROVED** [review]

**Comment by tdgroot** [comment]
You have some merge conflicts, can you update your branch? Then we can get this merged for release :)

**Comment by stijnbernards** [comment]
> You have some merge conflicts, can you update your branch? Then we can get this merged for release :)

Done :)

**Review by tdgroot — APPROVED** [review]

**Pull request merged**